include/core/jbackend.h

```diff
@@ -45,7 +45,7 @@ G_BEGIN_DECLS
 // FIXME does it make sense to report these errors?
 enum JBackendBSONError
 {
-	J_BACKEND_BSON_ERROR_BSON_APPEND_ARRAY_FAILED,
+	J_BACKEND_BSON_ERROR_BSON_APPEND_ARRAY_FAILED = 1,
```

Comment:
Isn't `FAILED` already a generic error?

Reply:
APPEND_ARRAY_FAILED is a specific error that appending to an array failed